avatar-and-badge/source/avatar.css

```diff
@@ -0,0 +1,133 @@
+@import url("./badge.css");
+@import url("/Alert-Button/source/alert.css");
+
+
+:root {
+  
+  --white-temp:rgb(255, 255, 255);
+  --online-status-color:rgb(59,165,93);
+  --dnd-status-color:#ed4245;
+  --offline-status-color:rgb(161, 159, 159);
+  --generic-transition:0.3s all ease-in;
+
+}
```

Comment:
Just a suggestion, if we could keep root css in the default css & then you can use it everywhere.

Reply:
Ohk, thanks for the suggestion🌟